app/components/nypr-brick-layout.js

```diff
@@ -0,0 +1,13 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+  tagName: '',
+  groups: Ember.computed('items', function() {
+    let items = this.get('items');
+    return [
+      items.slice(0, 3),
+      items.slice(3, 6),
+      items.slice(8)
```

Comment:
If we dont' want to stick to just 3 groups, this can be refactored to
```javascript
let groups = []
let items = this.get('items')
for (let i = 0; i < items.length; i += 3) {
  groups.push(items.slice(i, i + 3));
}
return groups;
```